example/res/xml/payment_sheet_preferences.xml

```diff
@@ -8,16 +8,21 @@
         app:title="Set merchant name"
         app:defaultValue="Widget Store" />
 
-    <SwitchPreferenceCompat
-        app:key="setup_intent"
-        app:title="Use SetupIntent"
-        app:defaultValue="false" />
+<!--    <SwitchPreferenceCompat-->
+<!--        app:key="setup_intent"-->
+<!--        app:title="Use SetupIntent"-->
+<!--        app:defaultValue="false" />-->
 
     <SwitchPreferenceCompat
         app:key="enable_customer"
         app:title="Use Customer API"
         app:defaultValue="true" />
 
+    <SwitchPreferenceCompat
+        app:key="returning_customer"
+        app:title="Returning Customer"
+        app:defaultValue="true" />
+
```

Comment:
Is there a way to consolidate this into a single pref, like "guest", "new customer", "returning customer"?